index.html

```diff
@@ -1 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
`The <h1>, <h2>, <p>, and <ul> elements are outside of the <body> element. They should be properly nested within the <body> to conform to standard HTML structure`